### DISCUSSION TITLE
Mirrors for x86_64, aarch64 can have different addresses.

### DISCUSSION THREAD

**Discussion by bwrsandman** [nroi/flexo#117] [Q&A] [answered]
ArchLinux and ArchARM all use pacman but they have different mirrors.
Is there a way to have the same flexo instance cache for both architectures and maintain separate mirror lists/rankings?

This probably also applies to msys2 and manjaro's different mirrors.

**Comment by nroi** [comment, marked as answer]
> Is there a way to have the same flexo instance cache for both architectures

No, this is not possible at the moment. I'm also not planning to implement this anytime soon.

The only workaround you can use is to set the `mirrors_predefined` variable instead of having the automatic mirror selection, and then set an ArchLinux ARM mirror there. But then, the flexo instance can be used for ARM clients only, not for x86.

A similar topic is also discussed [here](https://github.com/nroi/flexo/issues/97).
